main.py

```diff
@@ -1,10 +1,49 @@
+import json
+
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as f:
+        players_data = json.load(f)
+
+    for nickname, player_data in players_data.items():
+        race_name = player_data["race"]["name"]
+        race_description = player_data["race"].get("description", "")
+        race, _ = Race.objects.get_or_create(
+            name=race_name,
+            defaults={"description": race_description}
+        )
+
+        for skill_data in player_data["race"].get("skills", []):
+            skill_name = skill_data["name"]
+            skill_bonus = skill_data["bonus"]
+            Skill.objects.get_or_create(
+                name=skill_name,
+                race=race,
+                defaults={"bonus": skill_bonus}
+            )
+        guild_data = player_data.get("guild")
```

Comment:
you can refactor this block without an `else`. First you can set guild as None and then use only one `if` to reset Guild if there is some data for it in json. Plus you can use walrus operator in your `if` statement.